.prettierignore

```diff
@@ -1,4 +1,3 @@
 coverage/**/*.js
-tests/**/*.sol
-tests_config/**/*.js
```

Comment:
should we add `tests/config` to this file too?

Reply:
On this folder I added the `.prettierrc` with the default that prettier uses so we still benefit from using prettier on the small changes that we make here for us while keeping their code untouched.

Reply:
eslint on the other hand should ignore this folder